packages/cli/src/index.ts

```diff
@@ -25,13 +25,26 @@ program
 program
   .command('encrypt')
   .description('Encrypt and share a secret')
-  .argument('<content>', 'Content to encrypt')
+  .argument('[content]', 'Content to encrypt')
+  .option('-f, --file <file>', 'Path to file to encrypt')
   .option('-p, --password <password>', 'Password to encrypt with')
   .option('-t, --ttl <duration>', 'Time to live (e.g., 1h, 1d)', '1h')
   .option('-b, --burn', 'Burn after reading', true)
   .option('--ip <ip>', 'Restrict access to specific IP address')
   .option('-r, --reads <count>', 'Number of times the secret can be read', undefined)
   .action(async (content, options) => {
+    if (options.file && !content) {
+      try {
+        content = await readFileSync(options.file, 'utf-8');
+      } catch (error) {
+        console.error(chalk.red(error instanceof Error ? error.message : 'Unknown error'));
+        process.exit(1);
+      }
+    } else {
+      console.error(chalk.red('Cannot provide both content and file'));
+      process.exit(1);
+    }
```

Comment:
This if else condition does not align with the intended behavior. It _only_ handles files. We need something along the following:

```ts
  if (options.file && content) {
    // error
  }

  if (options.file) {
    // handle file
  } else {
    // handle content
  }
```